Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/UserDatasetEventHandler.java

```diff
@@ -1,21 +1,26 @@
 package org.gusdb.wdk.model.user.dataset.event;
```

Comment:
The diff on this file is a bit wonky as most of the contents of the file were moved to UserDatasetEventSyncHandler.